build-config/spotbugs-exclude.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ /*
+  ~ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  ~ */
+  -->
+
+<FindBugsFilter>
+
+    <Match>
+        <Class name="io.ballerina.persist.cmd.Push" />
+        <Method name="execute" />
+        <Bug pattern="SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING" />
+    </Match>
+
+</FindBugsFilter>
```

Comment:
```suggestion
</FindBugsFilter>

```

Reply:
Updated in [d8e10fa](https://github.com/ballerina-platform/persist-tools/pull/76/commits/d8e10fa12989ebdfdbfd1eb0d8c22b14965fcd08)